charts/talks/templates/ingress.yaml

```diff
@@ -0,0 +1,37 @@
+{{- $serviceName := include "fullname" . -}}
+{{- $servicePort := .Values.service.externalPort -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- range $key, $value := .Values.ingress.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+    {{- if .Values.ingress.globalStaticIpName }}
+      kubernetes.io/ingress.global-static-ip-name: {{ .Values.ingress.globalStaticIpName }}
+    {{- end }}
+spec:
+  rules:
+    {{- range $host := .Values.ingress.hosts }}
+    - host: {{ $host }}
+      http:
+        paths:
+          - path: /*
+            backend:
+              serviceName: {{ $serviceName }}
+              servicePort: {{ $servicePort }}
+    {{- end -}}
+  {{- if .Values.ingress.tls }}
+  tls:
+    - secretName: "{{ template "fullname" . }}-tls-secret"
```

Comment:
It actually does create it. I've deployed again the talks page with a new secret name (removing the `-secret`) part. A new certificate request has succeeded

```
time="2017-08-23T16:44:39Z" level=info msg="requesting certificate for talks.srcd.run" context="ingress_tls" name=test-talks-talks namespace=default
time="2017-08-23T16:44:55Z" level=info msg="authorization successful" context=acme domain=talks.srcd.run
time="2017-08-23T16:44:56Z" level=info msg="successfully got certificate: domains=[talks.srcd.run] url=https://acme-staging.api.letsencrypt.org/acme/cert/fa01e91c13ee3bfec4f2423fdf9f9fd0c600" context=acme
time="2017-08-23T16:44:56Z" level=info msg="Attempting to create new secret" context=secret name=test-talks-talks-tls namespace=default
time="2017-08-23T16:44:56Z" level=info msg="Secret successfully stored" context=secret name=test-talks-talks-tls namespace=default
```